app/services/repository_service.py

```
@@ -0,0 +1,39 @@
+from typing import List, Tuple
+
+from tortoise.transactions import in_transaction
```

Ruff (0.11.9)

`tortoise.transactions.in_transaction` imported but unused

Remove unused import: `tortoise.transactions.in_transaction`

(F401)